views/js/previewer/plugins/content/scoringHighlighter.js

```diff
@@ -105,15 +138,35 @@ define([
             this.hide();
 
             const $eraser = $container.find('button.icon-eraser');
+            const $color = $container.find('.color-button');
+            let isEraserOn = false;
+
+            const toggleEraser = (isEraserOn) => {
```

Comment:
We have both `toggleEraserMode` and `toggleEraser` functions, it can be a bit confusing I guess. Can we combine them into one?

Reply:
agreed!